### ISSUE TITLE
Castbar Player doesn't render properly after a reload UI, with the option "Cast Bar Underneath"

### ISSUE THREAD

**Issue by CaffeinatedCake** [wardz/ClassicCastbars#21]
_Bug reproduced in v1.1.1 and v1.1.2_

**What is the problem? Is there any Lua errors?**
The option "Cast Bar Underneath" doesn't work properly after a reload UI. The cast bar stops being properly rendered in the interface. There's no Lua error.

**What steps will reproduce the problem?**
After activating the castbar for the player, which works perfectly fine, I activate the option "Cast Bar Underneath":
![WoWScrnShot_102319_211440](https://user-images.githubusercontent.com/360734/67436517-db31b400-f5ee-11e9-99c3-14d587c42ac8.jpg)

Once it's done, it's working fine:
![WoWScrnShot_102319_211457](https://user-images.githubusercontent.com/360734/67437015-1ed8ed80-f5f0-11e9-96d5-b534708b1a3c.jpg)

But if you restart the game, or even simply do a reload UI, this is what you end up with:
![WoWScrnShot_102319_211526](https://user-images.githubusercontent.com/360734/67437051-3dd77f80-f5f0-11e9-9698-bd32d754185a.jpg)

At that point, it even shows on the "Toggle Testing Player" bar:
![WoWScrnShot_102319_211552](https://user-images.githubusercontent.com/360734/67437196-91e26400-f5f0-11e9-886c-1fe39b0bca1b.jpg)

If I just disable the option "Cast Bar Underneath" after that, this is what I get:
![WoWScrnShot_102319_211610](https://user-images.githubusercontent.com/360734/67437125-68293d00-f5f0-11e9-9851-52d70ee871d5.jpg)

Once the option is disabled and the interface reloaded, I'm back to normal:
![WoWScrnShot_102319_211624](https://user-images.githubusercontent.com/360734/67437147-76775900-f5f0-11e9-9de4-d936c92aa881.jpg)

**Does the issue still occur when all addons except ClassicCastbars are disabled?**
Yes, it does.

**Was it working in a previous version? If yes, which was the last good one?**
I started using ClassicCastbars from the v1.1.1, and the bug already existed at that point. I don't know about older versions.

**Comment by wardz** [comment]
Thanks for reporting, fixed it now. Will upload the new version tomorrow most likely.

PS: You'll have to manually change the castbar size when enabling castbar undearneath option if you want it small, since the persistent config is set to same size when using non underneath bar.

**Comment by CaffeinatedCake** [comment]
Great news! And I don't mind the extra step to get it to the right size. The config is pretty easy anyway, this is why I like this addon.

I will most likely test that this evening, after work.

**Comment by CaffeinatedCake** [comment]
Alright, I just did my test (I took the file you modified and use it in the current version, because I'm patient like that). It's all good!

Side note, the size options don't seem to want to use the round values I type :') (but I don't really care, 0.1 is not something I'll catch by eyes on my castbar)
![Capture](https://user-images.githubusercontent.com/360734/67510066-93fdfe80-f694-11e9-8676-f18ba2ef76b2.PNG)

A quick suggestion for a possible future version would be to add a reset button for the castbars dimensions, so people don't complain about not remembering the default values.

Anyway, thanks for the fix and your work on this addon!
Have a great day!/night!
